src/render/renderer.zig

```diff
@@ -22,6 +22,23 @@ pub const Renderer = extern struct {
         return wlr_renderer_autocreate(backend) orelse error.RendererCreateFailed;
     }
 
+    /// Since the impl of wlr_egl is not exposed in wlroots headers, we refer to it as opaque for ABI compatibility.
+    /// opaque: struct wlr_egl;
+    extern fn wlr_gles2_renderer_create(egl: *opaque {}) ?*Renderer;
+    pub fn createGles2(egl: *opaque {}) !*Renderer {
+        return wlr_gles2_renderer_create(egl) orelse error.wlr.RendererCreateFailed;
+    }
```

Comment:
This doesn't look like it will compile... how have you tested this?

Reply:
I generally expect patches adding additional bindings to be motivated by a concrete use-case in some project. A good way to test this would be to get whatever your end goal is working before upsteaming your zig-wlroots additions.

As for why zig build test passes, it's probably related to the fact that these additions aren't even possible to use currently as the aren't exposed in wlroots.zig